app/src/main/java/com/example/criminalintentapp/presentation/authentication/AppModule.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.criminalintentapp.presentation.authentication
```

Comment:
Small improvement: naming it `AppModules` might be better, since we have at least 2 modules